code/__DEFINES/icon_smoothing.dm

```diff
@@ -57,8 +57,8 @@ DEFINE_BITFIELD(smoothing_flags, list(
 #define SMOOTH_GROUP_CARPET_ROYAL_BLACK S_TURF(18)		///turf/open/floor/carpet/royalblack
 #define SMOOTH_GROUP_CARPET_ROYAL_BLUE S_TURF(19)		///turf/open/floor/carpet/royalblue
 #define SMOOTH_GROUP_CARPET_GRIMY S_TURF(20)			///turf/open/floor/carpet/grimy
-#define SMOOTH_GROUP_BAMBOO_FLOOR S_TURF(52) 			//![/turf/open/floor/bamboo]
-#define SMOOTH_GROUP_CATWALK  S_OBJ(31) 				///obj/structure/lattice/catwalk
+#define SMOOTH_GROUP_BAMBOO_FLOOR S_TURF(21) 			//![/turf/open/floor/bamboo]
+#define SMOOTH_GROUP_CATWALK  S_OBJ(22) 				///obj/structure/lattice/catwalk
```

Comment:
21,22....24?

Reply:
Quoting from Marvin  (Wanders, 0.1, MACro) over from discord:

> I think I figured it out. the Smooth Group for bamboo is number 52, while the "maximum" for turf is SMOOTH_GROUP_SURVIVAL_TITANIUM_WALLS  at a cool number 30. Glass is OBJ(21), so 30 + 1 + 21 is 52, the same group as bamboo floors.
The 52 should just probably be 21, and the catwalk below it should probably be 22